app/javascript/forms/mappers/validatorMapper.jsx

```diff
@@ -13,10 +13,16 @@ const fileValidator = ({ maxSize }) => {
     const imageTypes = /image\/jpg|image\/jpeg|image\/png|image\/svg/;
 
     if (value && value.inputFiles[0] && !imageTypes.test(value.inputFiles[0].type))
-      return __(`File must be an image of type "png", "jpg/jpeg", or "svg". The currently uploaded file's extension is "${value.inputFiles[0].type.split('/').pop()}"`);
+      return sprintf(__(`File must be an image of type "png", "jpg/jpeg", or "svg". The currently uploaded file's extension is "%s"`), value.inputFiles[0].type.split('/').pop());
 
     if (value && value.inputFiles[0] && value.inputFiles[0].size > maxSize)
-      return __(`File is too large, maximum allowed size is ${maxSize} bytes. Current file has ${value.inputFiles[0].size} bytes`);
+      return sprintf(
+        n__(
+          `File is too large, maximum allowed size is %s byte. Current file has %s bytes`,
+          `File is too large, maximum allowed size is %s bytes. Current file has %s bytes`, 
+          value.inputFiles[0].size
+        ), 
+        maxSize, value.inputFiles[0].size);
```

Comment:
maxSize will always be huge, and thus plural...think this should be:

```suggestion
      return sprintf(
        n__(
          `File is too large, maximum allowed size is %s bytes. Current file has %s byte`,
          `File is too large, maximum allowed size is %s bytes. Current file has %s bytes`, 
          value.inputFiles[0].size
        ), 
        maxSize, value.inputFiles[0].size);
```